netlify.toml

```diff
@@ -1,79 +1,97 @@
-      script-src-elem 'self' 'unsafe-eval' 'unsafe-inline' https://cdn.lr-ingest.io https://js.stripe.com https://netlify-cdp-loader.netlify.app/netlify.js https://prefect.my.salesforce.com/ https://prefect.force.com/ https://prefect.my.salesforce.com https://c.la1-c2-ia2.salesforceliveagent.com/ https://d.la1-c2-ia2.salesforceliveagent.com/ https://static.lightning.force.com https://prefect--partial.my.salesforce.com https://d.la3-c1cs-ia2.salesforceliveagent.com/ https://service.force.com/ https://prefect--aleysian.my.salesforce.com https://aleysian-prefect.cs29.force.com/ https://d.la1-c2-ph2.salesforceliveagent.com/ https://partial-prefect.cs1.force.com/ https://c.la1-c1cs-ph2.salesforceliveagent.com/ https://d.la1-c1cs-ph2.salesforceliveagent.com/ https://prefect--partial.sandbox.my.salesforce.com/ https://d.la2-c1-ia5.salesforceliveagent.com https://d.la3-c1cs-ia5.salesforceliveagent.com https://prefect--partial.sandbox.my.site.com; \
       img-src * data:; \
+      media-src 'none'; \
+      object-src 'none'; \
       style-src 'self' 'unsafe-inline' https://prefect.my.salesforce.com/ https://prefect.force.com/ https://prefect.my.salesforce.com https://c.la1-c2-ia2.salesforceliveagent.com/ https://d.la1-c2-ia2.salesforceliveagent.com/ https://static.lightning.force.com https://prefect--partial.my.salesforce.com https://d.la3-c1cs-ia2.salesforceliveagent.com/ https://service.force.com/ https://prefect--aleysian.my.salesforce.com https://aleysian-prefect.cs29.force.com/ https://partial-prefect.cs1.force.com/ https://c.la1-c1cs-ph2.salesforceliveagent.com/ https://d.la1-c1cs-ph2.salesforceliveagent.com/ https://prefect--partial.sandbox.my.site.com; \
-      style-src-elem 'self' 'unsafe-inline' https://prefect.my.salesforce.com/ https://prefect.force.com/ https://prefect.my.salesforce.com https://c.la1-c2-ia2.salesforceliveagent.com/ https://d.la1-c2-ia2.salesforceliveagent.com/ https://static.lightning.force.com https://prefect--partial.my.salesforce.com https://d.la3-c1cs-ia2.salesforceliveagent.com/ https://service.force.com/ https://prefect--aleysian.my.salesforce.com https://aleysian-prefect.cs29.force.com/ https://partial-prefect.cs1.force.com/ https://c.la1-c1cs-ph2.salesforceliveagent.com/ https://d.la1-c1cs-ph2.salesforceliveagent.com/ https://prefect--partial.sandbox.my.site.com; \ 
```

Comment:
I don't know that `style-src-elem` is beholden to the same inconsistent fallback pattern but without testing I'm wary of removing this since it could break the SF bot